scripts/check.sh

```diff
@@ -20,7 +20,7 @@ check-protos-compatible() {
     export PATH=$GOPATH/bin:$PATH
 
     if [ ! -f "$GOPATH/bin/protolock" ]; then
-        GO111MODULE=off go install github.com/nilslice/protolock/cmd/protolock@v0.17.0
+        go install github.com/nilslice/protolock/cmd/protolock@v0.17.0 || exit 1
```

Comment:
By removing `GO111MODULE=off`, please increase the go requirement to `1.16` in go.mod.
https://go.dev/ref/mod#:~:text=Since%20Go%201.16%2C%20go%20install%20is%20the%20recommended%20command